### PR TITLE
added board config files

### PR DESCRIPTION
**Pull request by opelpanfan**
small steps,

### CODE REVIEW

File: firmware/config/boards/core8/board.mk [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# List of all the board related files.
+BOARDCPPSRC =  $(PROJECT_DIR)/config/boards/core8/board_configuration.cpp
+
+BOARDINC = $(PROJECT_DIR)/config/boards/core8
+
+# Override DEFAULT_ENGINE_TYPE
+DDEFS += -DSTM32F407xx
```
Comment (mck1117) [inline firmware/config/boards/core8/board.mk:7]: probably don't need this line

Comment (opelpanfan) [reply firmware/config/boards/core8/board.mk:7]: MCU type or BOARDINC

File: firmware/config/boards/core8/board_configuration.cpp [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+	// TLE9201 driver
+	// This chip has three control pins:
+	// DIR - sets direction of the motor
+	// PWM - pwm control (enable high, coast low)
+	// DIS - disables motor (enable low)
+
+	// Throttle #1
+	// PWM pin
+	engineConfiguration->etbIo[0].controlPin = GPIOB_8;
```
[the remark this answers is not in the document]
Comment (opelpanfan) [reply firmware/config/boards/core8/board_configuration.cpp:130]: Sure make sense.

File: firmware/config/boards/core8/board.mk (hunk 1 above)
Comment (mck1117) [reply firmware/config/boards/core8/board.mk:7]: ` DDEFS += -DSTM32F407xx `

**Pull request merged**